### ISSUE TITLE
crash browser if usings screen=constrains 

### ISSUE THREAD

**Issue by DeathJoh** [daijro/camoufox#97]
### Describe the bug:
crash browser  
load page and next is crash browser after 1 secods 

### Screenshots:

![image](https://github.com/user-attachments/assets/49a44c60-2579-43ca-b6a4-fc9c957a17f3)

### To Reproduce:

this bug have onli if using screen in new version 

```
from browserforge.fingerprints import Screen
constrains = Screen(max_width=1920, max_height=1080)
screen=constrains
```

### Version:
Pip package:    v0.4.1
Camoufox:       v132.0.2-beta.17 (Up to date!)

![image](https://github.com/user-attachments/assets/e78e13f2-f314-4ca0-80cc-ad215b566f75)

**Comment by DeathJoh** [comment]
![image](https://github.com/user-attachments/assets/189b717d-73a3-4dca-803a-384107a12914)
 have next x x Crash no know why have this crash ? maibe if  Screen   have more to max_width=1920, max_height=1080

**Comment by daijro** [comment]
The crash is likely related to the WebGL fingerprint rotation that was just added earlier, and rolled back about half an hour ago.

Try updating to 0.4.2: `pip install -U camoufox`

Or enabling `block_webgl=True`.

**Comment by daijro** [comment]
Fixed in 0.4.4 :+1:
